src/Sulu/Bundle/WebsiteBundle/Resources/views/Sitemap/sitemap.xml.twig

```diff
@@ -1,42 +1,15 @@
 {% spaceless %}
 <?xml version="1.0" encoding="UTF-8"?>
-<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9" xmlns:xhtml="http://www.w3.org/1999/xhtml">
+<urlset
+        xmlns:xhtml="http://www.w3.org/1999/xhtml"
+        {% include 'SuluWebsiteBundle:Sitemap:namespaces.xml.twig' %}
```

Comment:
would just add a block instead of an own twig template!

Reply:
see: https://symfony.com/blog/new-in-symfony-3-4-improved-the-overriding-of-templates